### ISSUE TITLE
Catalyst: missing author in chat messages

### ISSUE THREAD

**Issue by ingoj** [anurodhp/Monal#304]
The author of messages (who sent which message) seems to be missing in Chats. That's make it hard to follow chats: 
<img width="457" alt="Bildschirmfoto 2019-12-20 um 20 49 23" src="https://user-images.githubusercontent.com/18173881/71309311-cca52480-2406-11ea-9e4d-2aa49f785ef2.png">

The author, however, is mentioned within the notifications that popup when Monal receives the new message, but missing in the chat window.

**Comment by Echolon** [comment]
Isn't that your text?

Is that in MUC?

**Comment by ingoj** [comment]
Nope, that text was from Anu. I have no idea why some of the messages appear twice, though...

**Comment by licaon-kter** [comment]
@ingoj the duplicate messages are another known issue

**Comment by anurodhp** [comment]
fixed in latest beta.
